app/controllers/shipit/shipit_controller.rb

```diff
@@ -16,6 +16,7 @@ class ShipitController < ApplicationController
       :ensure_required_settings,
       :force_github_authentication,
       :set_variant,
+      :check_github_status,
```

Comment:
Last minute though, IMO instead of a before_action that set something in the flash. We'd be better to render a banner in the layout.